packages/apollo-server/src/utils/exports.ts

```diff
@@ -0,0 +1,6 @@
+export * from 'graphql-tools';
+export * from 'graphql-subscriptions';
```

Comment:
Likewise to `graphql-tools`, I know it would require revisiting in the event of a change to `graphql-subscriptions`, but can we also be explicit here to avoid clashing?